.circleci/config.yml

```diff
@@ -586,6 +586,7 @@ jobs:
         type: string
         default: ""
     executor: << parameters.e >>
+    parallelism: 4
```

Comment:
I don't think the `parallelism` parameter works for Docker executors? IIRC it's only for `machine` executors

Reply:
Not sure if it actually works. I don't see 4 machines running it https://app.circleci.com/pipelines/github/metabase/metabase?branch=cypress-parallel

(separately there's another issue with it being confused by two sets of cypress tests)